code/drasil-example/ssp/lib/Drasil/SSP/DataDefs.hs

```diff
@@ -48,7 +48,9 @@ intersliceWtrFEqn = completeCase [case1,case2,case3]
           sy waterWeight), inxi waterHght $>= inxi slopeHght)
 
         case2 = (half (square (inxi waterHght $- inxi slipHght))  `mulRe` sy waterWeight,
-                inxi slopeHght $> inxi waterHght $> inxi slipHght)
```

Comment:
For later, we can probably make this a syntactic sugar of *Expr. I will open a ticket referencing this.